src/server/routes/pdfs.route.js

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
There is no corresponding test module to this

Reply:
corresponding test module now added